assembly-looknfeel-css.xml

```diff
@@ -0,0 +1,25 @@
+<assembly>
+    <id>css</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.build.directory}/css</directory>
+            <outputDirectory></outputDirectory>
+            <filtered>true</filtered>
+            <includes>
+                <include>**/*.html</include>
+                <include>VERSION</include>
+            </includes>
+        </fileSet>
```

Comment:
I'm not sure that this is required (the homepage and the version files are only in bonita-web look'n'feel, I think).
By the way, we could put the version only in distrib.